.dotnet/src/Custom/Chat/ChatMessage.cs

```diff
@@ -58,7 +58,7 @@ public abstract partial class ChatMessage
     /// <summary>
     /// The content associated with the message. The interpretation of this content will vary depending on the message type.
     /// </summary>
-    public IList<ChatMessageContentPart> Content { get; protected init; }
+    public IList<ChatMessageContentPart> Content { get; protected set; }
```

Comment:
This one is also a collection.

Reply:
Let's fix it in a separate PR - https://github.com/joseharriaga/openai-in-typespec/pull/182#discussion_r1693640061